bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/TupleValueImpl.java

```diff
@@ -827,4 +829,17 @@ private void resetSize(int index) {
             size = index + 1;
         }
     }
+
+    private void validateInherentTypeOfExistingMembers(long index, int offset) {
+        Type targetType;
+        for (long i = index; i < this.size; i++) {
+            targetType = i + offset >= this.tupleType.getTupleTypes().size() ?
```

Comment:
```suggestion
            targetType = (i + offset >= this.tupleType.getTupleTypes().size()) ?
```
This may improve readability.

Reply:
Thank you. I will add this.